charts/v3.0/cray-hms-hbtd/Chart.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
     version: "~10.0"
     repository: https://artifactory.algol60.net/artifactory/csm-helm-charts
   - name: cray-etcd-base
-    version: "~1.0"
+    version: "~1.1.0"
```

Comment:
Do you know if there is a difference between ~1.1 and ~1.1.0?

Will patch changes to the etcd-base chart get properly picked up when using ~1.1.0?

Reply:
Thanks, for pointing this out. I'll change it to "~1.1", since that is the previous convention in our code.
My reading of the docs is that:
"~1.1" is the same as "~1.1.0".
"~1.1.1" would be almost the same except for the higher allowed lower value.
see https://github.com/Masterminds/semver#tilde-range-comparisons-patch